test/source/browser/controllable.ts

```diff
@@ -523,6 +523,11 @@ abstract class ControllableBase {
     return files;
   };
 
+  public checkElementCount = async (selector: string, expectedCount: number) => {
```

Comment:
There is already [ensureElementCount](https://github.com/FlowCrypt/flowcrypt-browser/blob/feature/issue-4282/test/source/browser/controllable.ts#L464) method which performs the same check. Let's merge them into one to remove duplication

Reply:
You are right 👍 
Just merged it